client/src/redux/shopCart/cart-selectors.js

```diff
@@ -0,0 +1,33 @@
+import { createSelector } from "reselect";
+
+const selectCart = state => state.cart;
+
+export const selectCartItems = createSelector(
+  [selectCart],
+  cart => cart.cartItems
+);
+
+export const selectCartHidden = createSelector(
+  [selectCart],
+  cart => cart.hidden
+);
+
+export const selectCartItemsCount = createSelector(
```

Comment:
reduce the total number of the products/items into a integer representation and display it in the cart icon